jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -1,10 +1,13 @@
 import os
 
 import numpy as np
+import asdf
```

Comment:
does it make sense to move `asdf` to the roman-optional dependencies as well?

Reply:
No, asdf is already a required dependency from a long time ago for JWST support.

https://github.com/spacetelescope/jdaviz/blob/6a2892c1c321547be7ea11756fadb086387a49b3/setup.cfg#L46